waku/v2/protocol/waku_relay.nim

```diff
@@ -19,9 +19,12 @@ const
 
 type
   WakuRelay* = ref object of GossipSub
+    confTopics*: seq[string] # Configured PubSub topics
```

Comment:
Conf seems like a concept that exists before we get here, maybe something like activeTopics? defaultTopics? topics?

Reply:
Yeah, perhaps `defaultTopics`! Can then add the `/default-waku/` to the same list and treat all of these equivalently.

Reply:
Done in 7bc5d0d